### DISCUSSION TITLE
Important Notice

### DISCUSSION THREAD

**Discussion by infinitepower18** [infinitepower18/WSA-SystemControl#22] [Announcements]
Following the [recent announcement](https://github.com/microsoft/WSA/discussions/536) by Microsoft to discontinue Windows Subsystem for Android, both WSA Sideloader and WSA System Control will no longer be getting any major feature updates apart from bug fixes until at least March 5, 2025. As development of WSA itself has ended, my efforts are best spent elsewhere.

I have not yet decided whether I will remove the apps from the Microsoft Store after March 5, 2025. Regardless of what I do with the store listings, please be rest assured that the two programs will continue to have all its releases and source code hosted on GitHub, with no plans to remove the GitHub repositories.

The WhatsApp and Telegram channels will also be discontinued after this date.

Thank you for your support over the past few years! WSA Sideloader was my first step into app development and it's been great working with the community.

**Comment by Philaupatte** [comment]
Hi,
Yes indeed, this is going to be closed and so I am looking for a replacement solution.

I try Bluestack and it works but I feel like using a all long train to cary out only a feather.

Have you any suggestion to recommand the best replacement solution?

**Reply by infinitepower18** [reply]
I find Bluestacks untrustworthy. I don't know if they still do this now but they install a crypto wallet application on the taskbar without the user's permission. It also force installs Bluestacks X even if you select only Bluestacks 5 on the website.

The only other emulator I can recommend is [Google Play Games](https://g.co/googleplaygames) for gaming purposes.
